doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -338,37 +338,37 @@ local html = import 'html.libsonnet';
         },
         {
           name: 'split',
-          params: ['str', 'c'],
+          params: ['str', 'pat'],
```

Comment:
1) IMO `sep` (separator) would be a better name than `pat`.
2) Technically changing the argument name is a breaking change. In case I don't think it's very reasonable to use it with a named argument, so it's fine. We need to be careful with that.

Reply:
I'm fine to leave as `c` if you think that's the right call. I only was thinking about how `std.findSubstr()` used "`pat`" rather than "`c`" and wanted to make it clearer that multi-character strings are supported. I hadn't even considered those who use named parameters...

Probably best to leave as `c` then, for sheer compatibility sake?

Reply:
Yeah, let's leave as `c`.